config-model-api/src/main/java/com/yahoo/config/model/api/ConfigChangeReindexAction.java

```diff
@@ -0,0 +1,20 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.config.model.api;
+
+import java.util.Optional;
+
+/**
+ * Represents an action to re-index a document type in order to handle a config change.
+ *
+ * @author bjorncs
+ */
+public interface ConfigChangeReindexAction extends ConfigChangeAction {
+
+    @Override default Type getType() { return Type.REINDEXING; }
+
+    /** @return name identifying this kind of change, used to identify names which should be allowed */
+    String name();
+
+    /** @return name of the document type that must bed re-indexed, or empty if all document types */
```

Comment:
Hmm, suggestions don't work today?

Just wanted to point out typo `bed --> be`

Reply:
Ah, no suggestions in per-commit-view, of course.